### PR TITLE
added script for subdomain finder

### LINKED ISSUE #723: Discovering Subdomains with Python
**Is your feature request related to a problem? Please describe.**
Discovering Subdomains with Python

**Details**
Python Script to List all the subdomains of a website and stored it in a CSV file.

### PR DESCRIPTION
**Pull request by tanishq-arya**
# Description

Fixes #723 

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [x ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ x] My code follows the style guidelines(Clean Code) of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works
Screenshot (1636)

### CODE REVIEW

**Comment by tanishq-arya** [comment]
@vybhav72954 Please review my PR

File: Python/Subdomain_Finder/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+### Subdomain Finder
+
+This Script is used to find the possible subdomains of a domain.
+
+### Domain name
+
+The url format for a domain name is: "http://subdomain.domain.top-level-domain"
```
Comment (vybhav72954) [inline Python/Subdomain_Finder/README.md:7]: This doesn't look right, for a starter keep this in either italics or backticks (the URL I mean)

File: Python/Subdomain_Finder/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+### Subdomain Finder
+
+This Script is used to find the possible subdomains of a domain.
```
Comment (vybhav72954) [inline Python/Subdomain_Finder/README.md:3]: Can we have a more detailed intro

File: Python/Subdomain_Finder/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+### Domain name
+
+The url format for a domain name is: "http://subdomain.domain.top-level-domain"
+
+### Setup
+
+1. Create a Virtual Environment.
+2. Install the requirements by using `pip3 install -r requiremnts.txt`
+3. Hurray.! You're ready to use the Script.
```
Comment (vybhav72954) [inline Python/Subdomain_Finder/README.md:13]: No need of the step 3

File: Python/Subdomain_Finder/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+### Prerequisites
+
+1. The available_subdomains.txt should be in the current directory whre the program exists.
```
Comment (vybhav72954) [inline Python/Subdomain_Finder/README.md:17]: available_subdomains.txt link this to the file

File: Python/Subdomain_Finder/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+### Running a file
+
+1. Run the script using `python3 subdomain_finder.py`
+
+2. When asked for domain, enter `your domain` . For example: wiki.com
+
+3. When asked for size, enter `size` . For example: 10 [Max-1000] 
+
+3. While the program is working you can view in the terminal:
+    + means a possible subdomain is found
```
Comment (vybhav72954) [inline Python/Subdomain_Finder/README.md:29]: PLease have a look at the rendered README, this doesn't look right at all

File: Python/Subdomain_Finder/README.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    . means the subdomain is not available
+
+### Sample Test Case
+
+Enter `wiki.com` when asked for input domain.
+
+Run the script and observe that output.csv file is created and it contains the possible subdomains.
+
+#### All the requirements for this script is mentioned in **requirements.txt** file.
```
Comment (vybhav72954) [inline Python/Subdomain_Finder/README.md:38]: Mention a list of DEpenedecy as well, above the setup

Comment (vybhav72954) [inline Python/Subdomain_Finder/README.md:38]: You can add a author section if you want

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
PLease correct the README. The URLs you have entered, check if its possible to make them appear like text and not links. Use Inline HTML for that. IIf you can achieve that, make sure they are in italics or backticks.

**Comment by vybhav72954** [comment]
I will have a look at the python file after you edit the README

**Comment by tanishq-arya** [comment]
Okay. Working on the changes. I'll update soon.

**Comment by tanishq-arya** [comment]
Okay. Working on the changes. I'll update soon.

**Comment by vybhav72954** [comment]
While you are at it, consider adding the path of the script to Python README as well.

File: Python/Subdomain_Finder/README.md (hunk 6 above)
Comment (tanishq-arya) [reply Python/Subdomain_Finder/README.md:38]: What information should I add in this section. Can you give give me some tips ?

**Comment by tanishq-arya** [comment]
@vybhav72954 I have updated the README. Also can you give me some suggestions as to what can I add in the Author section ?

**Comment by tanishq-arya** [comment]
@vybhav72954 any update on this. please merge this if everything is okay.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**